modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomValidation.testcase

```diff
@@ -21,56 +21,84 @@ definition {
 		}
 	}
 
-	@description = "LPS-144474 - Verify that, when adding a new entry in the Object, the user only is able to submit the entry successfully if it passes for all validations from that Object"
-	@priority = 4
+	@description = "LPS-144474 - Verify it's possible to add valid entries when define output fields for my validation as full validation"
+	@priority = 5
 	test CanAddEntry {
+		property custom.properties = "feature.flag.LPS-187846=true";
+		property portal.acceptance = "true";
```

Comment:
Let's not add this to acceptance yet since we want to ensure the test is stable first